packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -56,6 +57,15 @@ typedef ExpansionPanelCallback = void Function(int panelIndex, bool isExpanded);
 /// [ExpansionPanel] needs to rebuild.
 typedef ExpansionPanelHeaderBuilder = Widget Function(BuildContext context, bool isExpanded);
 
+/// Signature for the callback that's called when the expansion indicator of the
+/// [ExpansionPanel] needs to rebuild.
```

Comment:
The doc here should mention what the parameters are and some general guideline on how they should be used when building widget